Add list of supported sensors with API and SDF links to README (#511)
gazebosim/gz-sensors: README.md

| **Camera**                | [CameraSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1CameraSensor.html)                  | [SDF Camera](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_camera)                          | [camera.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/camera.cc)                            | Captures standard RGB images                       |
| **Depth Camera**          | [DepthCameraSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1DepthCameraSensor.html)        | [SDF Depth Camera](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_depth_camera)              | [depth_camera.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/depth_camera.cc)                | Captures depth information                         |

> | **Doppler Velocity Log**  | [DopplerVelocityLog](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1DopplerVelocityLog.html)      | [SDF DVL](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_dvl)                                | [dvl.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/dvl.cc)                                  | Measures velocity relative to a water body         |

Reviewer: the sensors produces velocity relative to seabed or by tracking water mass. I think we don't have to that detailed and just have a generic description note:

```suggestion
| **Doppler Velocity Log**  | [DopplerVelocityLog](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1DopplerVelocityLog.html)      | [SDF DVL](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_dvl)                                | [dvl.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/dvl.cc)                                  | Measures relative velocity of an underwater vehicle        |
```

| **Force-Torque**          | [ForceTorqueSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1ForceTorqueSensor.html)        | [SDF Force-Torque](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_force_torque)              | [force_torque.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/force_torque.cc)                | Measures forces and torques on a joint             |

> | **GPU Lidar**             | [GpuLidarSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1GpuLidarSensor.html)              | [SDF GPU Lidar](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_gpu_lidar)                    | [gpu_lidar_sensor.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/gpu_lidar_sensor.cc)                      | Simulates a 3D laser scanner using GPU acceleration|

Reviewer: update link:

```suggestion
| **GPU Lidar**             | [GpuLidarSensor](https://gazebosim.org/api/sensors/9/classgz_1_1sensors_1_1GpuLidarSensor.html)              | [SDF GPU Lidar](http://sdformat.org/spec?ver=1.9&elem=sensor#sensor_lidar)                    | [gpu_lidar_sensor.cc](https://github.com/gazebosim/gz-sensors/blob/main/test/integration/gpu_lidar_sensor.cc)                      | Simulates a 3D laser scanner using GPU acceleration|
```
